packages/node/src/extensions/error-tracking/autocapture.ts

```diff
@@ -50,13 +50,13 @@ export function addUncaughtExceptionListener(
   captureFn: (exception: Error, hint: CoreErrorTracking.EventHint) => void,
   onFatalFn: (exception: Error) => void
 ): void {
-  global.process.on('uncaughtException', makeUncaughtExceptionHandler(captureFn, onFatalFn))
+  getGlobalSafely()?.process.on('uncaughtException', makeUncaughtExceptionHandler(captureFn, onFatalFn))
```

Comment:
you can use globalThis instead of global. it is defined on node and edge runtime but process won't be defined on edge, so this should work :
```suggestion
  globalThis.process?.on('uncaughtException', makeUncaughtExceptionHandler(captureFn, onFatalFn))
```